agent-part/src/main/kotlin/Plugin.kt

```diff
@@ -45,6 +50,37 @@ class Plugin(
     private val instrumenter: DrillInstrumenter = instrumenter(instrContext, logger)
 
     private val _retransformed = atomic(false)
+    private var db: DB
+    val probesDb: HTreeMap.KeySet<Any>
+
+    init {
+        //TODO this is hack to initialize mapDB
+        Thread.currentThread().contextClassLoader = ClassLoader.getSystemClassLoader()
+        //TODO crash on agent restart without db clearing
+        db = DBMaker.fileDB("test2code.db")
+            .transactionEnable()
+            .closeOnJvmShutdown()
```

Comment:
[Tests fix] It may be necessary to delete files after closing db and disable file locking